manifests/install.yaml

```diff
@@ -7,4 +7,4 @@ metadata:
 spec:
   sources:
   - web:
-      url: https://github.com/argoproj-labs/rollout-extension/releases/download/v0.1.0/extension.tar
+      url: https://github.com/alexef/rollout-extension/releases/download/v0.2.2/extension.tar
```

Comment:
I think we'd like to keep this in the `argoproj-labs` repository

Reply:
sorry, I must have pushed on the wrong branch. let me revert this commit

Reply:
```suggestion
      url: https://github.com/argoproj-labs/rollout-extension/releases/download/v0.2.1/extension.tar
```